### PR TITLE
Don't show sticky newsletter sign up button if Tito widget is presented

### PR DESCRIPTION
**Pull request by mmmavis**
Don't show sticky newsletter sign up button if Tito widget is presented.

Related PRs/issues: #10307

- Go to http://mozfest.localhost:8000/en/newsletter/ on a smaller screen and verify there's no sticky button at page bottom
- Click on "Secure your ticket" from the nav bar. Make sure you are able to access full content in the Tito popup.
- - Click on "Secure your MozFest ticket" before the "Lastest Festival News" section. Make sure you are able to access full content in the Tito popup.

### CODE REVIEW

**Review by danielfmiranda — CHANGES_REQUESTED** [review]
Hi @mmmavis! 

This works great for the initial sign up form, however, once you follow the sign up flow to the "assign your tickets" page, the "Sign Up" button appears to cover the buttons at the bottom of the modal again:
Screenshot 2023-03-08 at 10 44 58 AM

File: source/js/foundation/pages/mozfest/tito.js [hunk 1]
```diff
@@ -1,17 +1,20 @@
 /**
- * Using the tito library to set up a ticket purchasing widget.
```
Comment (mmmavis) [inline source/js/foundation/pages/mozfest/tito.js:2]: This file didn't seem to be used/imported anywhere in the codebase. I'm reviving it.

**Comment by mmmavis** [comment]
@danielfmiranda thanks for catching the bug. I realized Tito API has something we could utilize so I went with that route instead of my original approach.

**Review by danielfmiranda — APPROVED** [review]
Thanks @mmmavis! The latest changes look good to me

**Pull request merged**